api/views.py

```diff
@@ -102,8 +95,10 @@ def events(self, request, pk=None):
 class WineView(viewsets.ModelViewSet):
     serializer_class = WineSerializer
 
+    permission_classes = [AllowWineryOwnerOrReadOnly]
+
     def create(self, request, winery_pk, wineline_pk):
-        serializer = WineSerializer(data=request.data)
+        serializer = serializer = self.get_serializer(data=request.data)
```

Comment:
this was a bug. When you call the serializer using get_serializer It put's in the serializer's context the request data.